### ISSUE TITLE
Create releases with precompiled binaries

### ISSUE THREAD

**Issue by lobsterdore** [andreas-schroeder/kafka-health-check#2]
It would be handy to have versioned releases with binaries for common platforms.

**Comment by starkers** [comment]
Seconded :+1: 

In the meantime: https://github.com/starkers/kafka-health-check/releases ;-)

**Comment by andreas-schroeder** [comment]
Hello, 

my apologies for being silent for quite some time.

I'll address this issue after reviewing the reported bugs :)

**Comment by andreas-schroeder** [comment]
see https://github.com/andreas-schroeder/kafka-health-check/releases/tag/v0.0.0

**Comment by jorgheymans** [comment]
the linux64 binaries don't work on my RHEL box, i'm not too knowledgable in go maybe i am missing something ?

```
[user@s-dev1]: ./kafha-health-check
./kafha-health-check: ./kafha-health-check: cannot execute binary file
```

**Comment by andreas-schroeder** [comment]
Hi @jorgheymans, I think you might have downloaded the binary for a different OS (and also, sorry for the typo, I've now fixed it in v0.0.1). Have you tried https://github.com/andreas-schroeder/kafka-health-check/releases/download/v0.0.1/kafka-health-check_0.0.1_linux_amd64.tar.gz ?

**Comment by jorgheymans** [comment]
you were right, no idea how i managed to mix up a simple download it's starting fine now!
